canonical-web-and-design/ubuntu.com
# Create Dell partner page and contact-form

## Done

- Created the /dell partner page
- Created the /dell#get-in-touch contact-us modal page
- Created the /dell/contact-us page
- Created the /dell/thank-you page
- Fixed a width in the /internet-of-things/appstore page

**NOTE** - *the reference architecture pages are not ready and the links hidden*

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: 
    - http://0.0.0.0:8001/dell
    - http://0.0.0.0:8001/dell#get-in-touch (fill in the form)
    - http://0.0.0.0:8001/dell/contact-us (fill in the form)
    - http://0.0.0.0:8001/dell/thank-you (you need to manually go there as the contact us will take you to ubuntu.com)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare /dell to the [copy doc](https://docs.google.com/document/d/1ZZQdZt2HFiLwjYYGsb-q7Zncdj0kdv0RBxFm7cz7Gu8/edit#)
- Compare /dell#get-in-touch to the [copy doc](https://docs.google.com/document/d/1tv_fsVebbIPgKjwnmawF6hlWTTPkvjE2jAjH3MBp5vU/edit#heading=h.tk4y6g6el1tm)

## Issue / Card

Fixes #6008
